component/LinkableText/LinkableText.tsx

```diff
@@ -2,11 +2,11 @@ import Link from "next/link";
 
 const LinkableText = ({text}: {text: string}) => {
     const sanitizeNewLine = text?.split('\n')
-    const regexUrl = /^((?:https?:\/\/)?[^./]+(?:\.[^./]+)+(?:\/.*)?)$/
+    const regexUrl = /(https?\:\/\/)?([^\.\s]+)?[^\.\s]+\.[^\s]+/gi
 
     const textComponent = sanitizeNewLine?.map((textline, index) => {
         const textLineSplit = textline?.split(' ')
-        return <p key={textline}>
+        return <p className="font-normal text-sm" key={textline}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a CSS class to the paragraph element will change its appearance. Ensure that this change is consistent with the overall design of your application. If you want to keep the old style, you can remove the added class.
```diff
- return <p className="font-normal text-sm" key={textline}>
+ return <p key={textline}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->